.travis.yml

```diff
@@ -0,0 +1,52 @@
+sudo: false
+dist: xenial
+addons:
+  apt:
+    sources:
+      - sourceline: "ppa:jonathonf/ffmpeg-4"
+    packages:
+      - libudev-dev
+      - libavformat-dev
+      - libavcodec-dev
+      - libavdevice-dev
+      - libavutil-dev
+      - libswscale-dev
+      - libswresample-dev
+      - libavfilter-dev
+matrix:
+  fast_finish: true
+  include:
+    - python: "3.5.3"
+      env: TOXENV=lint
+    - python: "3.5.3"
+      env: TOXENV=pylint
+    - python: "3.5.3"
+      env: TOXENV=typing
+    - python: "3.6"
```

Comment:
I suggest only left lowest support version and highest support version for personal use, i.e. 3.5.3 and 3.8-dev

Reply:
Since 3.8-dev failed in Travis, I picked 3.7 instead.